contrib/builder/rpm/amd64/amazonlinux-latest/Dockerfile

```diff
@@ -5,9 +5,9 @@
 FROM amazonlinux:latest
 
 RUN yum groupinstall -y "Development Tools"
-RUN yum install -y btrfs-progs-devel device-mapper-devel glibc-static libseccomp-devel libselinux-devel libtool-ltdl-devel pkgconfig selinux-policy selinux-policy-devel  tar git cmake vim-common
+RUN yum install -y btrfs-progs-devel device-mapper-devel glibc-static libseccomp-devel libselinux-devel pkgconfig selinux-policy selinux-policy-devel  tar git cmake vim-common
```

Comment:
the generate script was updated in https://github.com/moby/moby/commit/b877fc31c51baaf1ec42501ab366dd05a5b6bb21, but looks like the script wasn't run to update the dockerfiles